.ci/setup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+test_repo="github.com/clearcontainers/tests"
```

Comment:
Will we include this in `kata-containers/tests` repo?

Reply:
@bergwolf - yes, once this PR is approved and landed, I plan to add `checkcommits` (with the same configuration) to all the kata code repos.